android/gradle.properties

```diff
@@ -1,5 +1,5 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
-AndroidProject.signing=/Users/donataspuidokas/key.properties
+AndroidProject.signing=./key.properties
```

Comment:
just to be extra careful, I wanted to push this file out of project directory as it contains sensitive information

Reply:
The local `.gitignore` already contains a line which should avoid checking in the file:
```
...
# Remember to never publicly share your keystore.
# See https://flutter.dev/docs/deployment/android#reference-the-keystore-from-the-app
key.properties
...
```